src/tests/ftest/util/osa_utils.py

```diff
@@ -16,6 +17,14 @@
 from pydaos.raw import (DaosContainer, IORequest,
                         DaosObj, DaosApiError)
 
+try:
+    # python 3.x
+    import queue as queue
```

Comment:
(pylint-useless-import-alias) Import alias does not rename original package

Reply:
Do we need to resolve python3 lint errors now? Atleast based on Test group discussion looks like we have enabled more lint errors... This should not be an issue. Several scripts have this issue.